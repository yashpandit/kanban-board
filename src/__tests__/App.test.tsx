import { expect, it, describe } from "vitest";
import { render } from "@testing-library/react";
import App from "../App";

describe("App", () => {
  it("should match snapshot", () => {
    const { baseElement } = render(<App />);
    expect(baseElement).toMatchSnapshot();
  });
});
