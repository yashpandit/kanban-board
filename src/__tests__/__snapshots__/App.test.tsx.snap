// Vitest Snapshot v1

exports[`App > should match snapshot 1`] = `
<body>
  <div>
    <div>
      hi
    </div>
  </div>
</body>
`;
